_includes/v3.10/release-notes/v3.10.0-release-notes.md

```diff
@@ -0,0 +1,59 @@
+
+#### Advertisement Kubernetes Service ExternalIPs
+
+Calico now supports BGP advertisement of Kubernetes service ExternalIPs in addition to advertising ClusterIPs. Advertising service external IPs allows for more flexible routing architectures.
+Along with this feature, we moved the configuration of ClusterIPs into BGPConfiguration (along with ExternalIPs) and have deprecated the environment variable `CALICO_ADVERTISE_CLUSTER_IPS`.
+We recommend replacing the deprecated `CALICO_ADVERTISE_CLUSTER_IPS` with a corresponding default BGPConfiguration resource.
+
+For more information please see the [service advertisement how-to](https://docs.projectcalico.org/v3.10/networking/advertise-service-ips)
+
+ - Support sending BGP advertisements for Kubernetes Service External IPs that fall within a set of whitelisted CIDR blocks. [libcalico-go #1123](https://github.com/projectcalico/libcalico-go/pull/1123) (@tompntn)
+ - Advertise Whitelisted Service External IP's. [confd # 266](https://github.com/projectcalico/confd/pull/266) (@tompntn)
+ - Configuration of Kubernetes service Cluster IP advertisement through BGPConfiguration resource. [libcalico-go #1134](https://github.com/projectcalico/libcalico-go/pull/1134) (@caseydavenport)
+ - Support datastore-driven clusterIP updates. [confd # 272](https://github.com/projectcalico/confd/pull/272) (@caseydavenport)
+
+#### Bug fixes
+
+ - Fix returning of resp.Body.Close() error. [node #329](https://github.com/projectcalico/node/pull/329) (@xieyanker)
+ - Fix erroneous "Partial success" output. [calicoctl #2070](https://github.com/projectcalico/calicoctl/pull/2070) (@lmm)
+ - Fix logging to be independent of go mod versions. [libcalico-go #1127](https://github.com/projectcalico/libcalico-go/pull/1127) (@neiljerram)
+
+#### Other changes
+
+ - Update calico/node to use a version of calico/bird based on upstream BIRD v1.6.7. This allows us to make use of LLGR (long-lived graceful restart). [node #314](https://github.com/projectcalico/node/pull/314) (@neiljerram)
+ - Use a real confd version. [node #315](https://github.com/projectcalico/node/pull/315) (@rafaelvanoni)
+ - Add semaphore config. [node #324](https://github.com/projectcalico/node/pull/324) (@neiljerram)
+ - The default IP pool block size is now configurable. Use the `CALICO_IPV4POOL_BLOCK_SIZE` and `CALICO_IPV6POOL_BLOCK_SIZE` environment variables to configure the default pool block size. [node #346](https://github.com/projectcalico/node/pull/346) (@halfcrazy)
+ - The `CALICO_IPV4POOL_NODE_SELECTOR` and `CALICO_IPV6POOL_NODE_SELECTOR` environment variables were added for configuring default pool node selectors. [node #346](https://github.com/projectcalico/node/pull/346) (@tmjd)
+ - Add calicoctl `patch` command similar to `kubectl patch`. [calicoctl #2054](https://github.com/projectcalico/calicoctl/pull/2054) (@syscll)
```

Comment:
We might want to highlight the `calicoctl patch` command , actually. I think that's an exciting one.

Reply:
Agreed, done